cabot/metricsapp/tests/fixtures/metrics_series.yaml

```diff
@@ -0,0 +1,39 @@
+error: False
+error_message:
+error_code:
+raw: testing123
+data:
+  - series: prod.good.data
+    datapoints:
+      - [1387817760, 8.14908]
```

Comment:
our boss would ask for "milliseconds" here ... so I am not tagging him. Let us see if he finds outs ;-)